### PR TITLE
Release v3.1.0-rc1

### PR DESCRIPTION
**Pull request by therealyingtong**

### CODE REVIEW

File: src/deprecation.h [hunk 1]
```diff
@@ -8,10 +8,9 @@
 // Deprecation policy:
 // * Shut down 16 weeks' worth of blocks after the estimated release block height.
 // * A warning is shown during the 2 weeks' worth of blocks prior to shut down.
-static const int APPROX_RELEASE_HEIGHT = 840277;
-// static const int WEEKS_UNTIL_DEPRECATION = 16;
-// static const int DEPRECATION_HEIGHT = APPROX_RELEASE_HEIGHT + (WEEKS_UNTIL_DEPRECATION * 7 * 24 * 48);
-static const int DEPRECATION_HEIGHT = 973713;
+static const int APPROX_RELEASE_HEIGHT = 901000;
+static const int WEEKS_UNTIL_DEPRECATION = 16;
+static const int DEPRECATION_HEIGHT = APPROX_RELEASE_HEIGHT + (WEEKS_UNTIL_DEPRECATION * 7 * 24 * 48);
```
Comment (str4d) [inline src/deprecation.h:13]: This gives me an EoS halt height of 1030024, which is before the targeted Canopy mainnet activation height of 1046400.

Comment (daira) [reply src/deprecation.h:13]: Thanks for remembering to check this! :-)

File: doc/release-notes/release-notes-3.1.0-rc1.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+
+Wladimir J. van der Laan (3):
+      Add `-debuglogfile` option
+      test: Add test for `-debuglogfile`
+      doc: Update release notes for `-debuglogfile`
+
+Marshall Gaucher (1):
+      Add helpers for tapping and donating to the testnet faucet
+
+therealyingtong (22):
```
Comment (daira) [inline doc/release-notes/release-notes-3.1.0-rc1.md:217]: Non-blocking (and doesn't need to be done for the RC): add an alias `'therealyingtong': 'Ying Tong Lai'` to `author_aliases` in `zcutil/release-notes.py`.

**Comment by daira** [comment]
@zkbot r+

**Pull request merged**